apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/rubricSettingsModal.ejs

```diff
@@ -0,0 +1,276 @@
+    <form name="rubric-settings" method="POST" class="needs-validation"
+          data-max-points="<%= assessment_question.max_points %>">
+      <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+      <input type="hidden" name="__action" value="modify_rubric_settings">
+      <input type="hidden" name="modified_at" value="<%= locals.rubric_data?.modified_at %>">
+      <input type="hidden" name="use_rubric" value="true">
+      
+      <div class="modal-content">
+        <div class="modal-header bg-info text-light">
+          <h5 class="modal-title">Rubric settings</h5>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+        </div>
+        <div class="modal-body">
+          <% if (assessment_question.max_auto_points) { %>
+          <div class="form-row">
+            <div class="col-6">
+              <div class="form-check">
+                <label class="form-check-label">
+                  <input class="form-check-input js-replace-auto-points-input"
+                         name="replace_auto_points"
+                         type="radio" value="true" required
+                         data-max-points="<%= assessment_question.max_points %>"
+                         <% if (locals.rubric_data?.replace_auto_points ?? !assessment_question.max_manual_points) { %>checked<% } %>
+                         aria-describedby="rubric-settings-replace-auto-explanation">
+                  Apply rubric to total points (ignore auto points)
+                </label>
+              </div>
+            </div>
+            <div class="col-6">
+              <div class="form-check">
+                <label class="form-check-label">
+                  <input class="form-check-input js-replace-auto-points-input" name="replace_auto_points"
+                         type="radio" value="false" required
+                         data-max-points="<%= assessment_question.max_manual_points %>"
+                         <% if (!(locals.rubric_data?.replace_auto_points ?? !assessment_question.max_manual_points)) { %>checked<% } %>
+                         aria-describedby="rubric-settings-replace-auto-explanation">
+                  Apply rubric to manual points (keep auto points)
+                </label>
+              </div>
+            </div>
+          </div>
+          <small id="rubric-settings-replace-auto-explanation" class="form-text text-muted">
+            If the rubric is applied to total points, then a student's auto points will be ignored, and the rubric items will be based on the total points of the question (<%= assessment_question.max_points %> points).
+            <br/>
+            If the rubric is applied to manual points only, then a student's auto points are kept, and the rubric items will be added to (or subtracted from) the autograder results.
```

Comment:
I really like having lots of help text in this dialog, but it makes everything very busy (especially when you see the dialog for the first time). Could we shift the help text to popovers triggered by (?) icons at the end of different options? It would be fine if multiple popovers have the same help-text in them, such as for the positive/negative grading help text which applies to both radio options.

Reply:
I replaced the text with tooltips in 61330bb5f, let me know if this is clearer. I'm happy to add a circle-info icon as well, if you prefer that.

Reply:
I personally prefer explicit "(?)" buttons to click on to get help, where there is space to do it. I reserve tooltips for when there really isn't space (or the help buttons would clutter the interface too much). For example, in GH's comment editor they have tooltips on all the icons above the text area, which seems like a good use for tooltips. In this case I think we have enough space for help buttons.

Reply:
@nwalters512 thoughts about this?

Reply:
In case my experience was useful: I just opened up the rubric modal for the first time and was like "wow, we should have some explanatory text here", not realizing that it would appear when I hovered on inputs! I think an explicit trigger would be an improvement, but I also wonder if we should err on the side of not hiding it in the first place, a la this type of thing used throughout GitHub's settings:

<img width="778" alt="Screenshot 2023-05-17 at 14 44 25" src="https://github.com/PrairieLearn/PrairieLearn/assets/1476544/efdb67e5-291e-403b-a8b4-83fb6928592c">
<img width="598" alt="Screenshot 2023-05-17 at 14 44 05" src="https://github.com/PrairieLearn/PrairieLearn/assets/1476544/463ec5ff-e199-49c4-8d2b-b23439fba616">

IMO we've misused popovers pretty frequently throughout PL's UI, and I think "just show it to people" would be an improvement here.

Reply:
Jonatan's first version actually had the text explicitly written everywhere (like the GH example above). I found it to be rather overwhelming in a dialog. I'm on board with the explicit text for things like settings pages where we are infrequently accessing them and we really want information about the options. This dialog is going to be used frequently by people who will know which options they want after the first few times using the dialog. We should certainly come up with some better guidelines for our use of help text.

@jonatanschroeder I guess you should go with whatever you think here (but probably not tooltips). Either your original design with the text just in the dialog or something with explicit popover triggers seem reasonable to me (with my vote going for "(?)" buttons).

Reply:
Would an ℹ️ with a tooltip (instead of a popover) work?

Reply:
Yep, works for me!

Reply:
Done in 489180211. Let me know if you want to change the icon or colour.

<img width="278" alt="image" src="https://github.com/PrairieLearn/PrairieLearn/assets/1004907/7b2b4dd7-b57d-421b-b335-57cd03def083">